stable/chaoskube/templates/deployment.yaml

```diff
@@ -73,7 +73,9 @@ spec:
 {{- if .Values.priorityClassName }}
       priorityClassName: "{{ .Values.priorityClassName }}"
 {{- end }}
-      serviceAccountName: {{ if .Values.rbac.create }}{{ printf "%s-%s" .Release.Name .Values.name }}{{ else }}"{{ .Values.rbac.serviceAccountName }}"{{ end }}
+{{- if .Values.rbac.create }}
+      serviceAccountName: "{{ .Values.rbac.serviceAccountName }}"
```

Comment:
When someone sets `rbac.create=false` and `rbac.serviceAccountName=foobar` then it seems they would get the `default` service account instead of the desired `foobar`.

Reply:
@jpds, could you address the comment by @linki?